open_diloco/run_training.sh

```diff
@@ -1,33 +1,47 @@
 #!/bin/bash
 
 ## example usage
-# ./run_training.sh 2 /ip4/127.0.0.1/tcp/36593/p2p/12D3KooWEAyutJ1zFqhAbzDn1LSzraB3o1uS8GSHxQYM87QP4AHN --per-device-train-batch-size 16 --batch-size 512 --local-steps 10  --total-steps 88000 --c4-tiny
+# ./run_training.sh 2 1 /ip4/127.0.0.1/tcp/36593/p2p/12D3KooWEAyutJ1zFqhAbzDn1LSzraB3o1uS8GSHxQYM87QP4AHN --per-device-train-batch-size 16 --batch-size 512 --local-steps 10  --total-steps 88000 --c4-tiny
```

Comment:
Maybe have a 2 x 2 example and say that this uses 4 gpu total so its more clear that this is the intended arg usage.